SpatialGDK/Source/SpatialGDK/Private/Interop/ActorGroupManager.cpp

```diff
@@ -129,6 +129,12 @@ void UActorGroupManager::ValidateOffloadingSettings(TMap<FName, FActorClassSet>
 		}
 	}
 
+	// Check for invalid WorkerType (Currently just UnrealClient)
+	if (WorkerTypes->Contains(FName(*SpatialConstants::ClientWorkerType)))
```

Comment:
Why does this need to be invalid? If actor groups are designed with this restriction in mind (because of RPC semantics, etc.), perhaps it's worth at least renaming `WorkerTypes` to `ServerTypes` and making this clear in the UI, too.